lib/boot-record.coffee

```diff
@@ -81,5 +81,5 @@ exports.getExtended = (image, position) ->
 #	bootRecord.getMaster('path/to/rpi.img').then (mbr) ->
 #		console.log(mbr)
 ###
-exports.getMaster = (image) ->
+exports.getMaster = getMaster = (image) ->
```

Comment:
I think this can be removed, since `getMaster` is not being used anywhere in this file.